x-pack/solutions/security/plugins/security_solution/public/flyout/attack_details/context.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { createContext, memo, useContext, useMemo } from 'react';
+import { useAssistantContext } from '@kbn/elastic-assistant';
+import type { AttackDetailsProps } from './types';
+import type { AttackDiscoveryItem } from './hooks/use_find_attack_details_by_id';
+import { useFindAttackDetailsById } from './hooks/use_find_attack_details_by_id';
+import { FlyoutLoading } from '../shared/components/flyout_loading';
+import { FlyoutError } from '../shared/components/flyout_error';
+
+export interface AttackDetailsContext {
+  /**
+   * Id of the document
+   */
+  documentId: string;
```

Comment:
I know I had said to use `documentId` here, but I think it actually makes more sense to use `attackId` here. `documentId` made sense for the `document_details` flyout as we could have alert or event documents...